grpc/interceptors_test.go

```diff
@@ -221,3 +238,116 @@ func TestRequestTimeTagging(t *testing.T) {
 	count := test.CountHistogramSamples(si.metrics.rpcLag)
 	test.AssertEquals(t, count, 1)
 }
+
+// blockedServer implements a ChillerServer with a Chill method that:
+//   a) Calls Done() on the received waitgroup when receiving an RPC
+//   b) Blocks the RPC on the roadblock waitgroup
+// This is used by TestInFlightRPCStat to test that the gauge for in-flight RPCs
+// is incremented and decremented as expected.
+type blockedServer struct {
+	roadblock, received *sync.WaitGroup
```

Comment:
Either the `WaitGroup`s or the `blockedServer` need to be pass-by-reference, not both. In this instance, since you're passing `blockedServer` by reference everywhere, I think it would be slightly more idiomatic to make the `sync.WaitGroup` a non-pointer.